### ISSUE TITLE
Ped animation problem.

### ISSUE THREAD

**Issue by pat225** [MAFINS/MenyooSP#412]
Hello, can you help me with my problem? I have problem with animations. I spawn ped, I set animation, example: amb@world_human_vehicle_mechanic@male@base and ped is doing mechanic animation, now, I click on Managed Saved Files and than I click on Save Database To File. I have this scene saved. Now, when I click on Load Placements, my scene is spawned, but ped is still standing on place without setted animation, in this case, amb@world_human_vehicle_mechanic@male@base. Animation doesn´t work on ped, after, Load placements. Can you help me, why, I have this problem and if I can repair it? Thanks for help

**Comment by OmegAlex10** [comment]
Is the "automatically play animations" toggled on? At your save file settings there should be that option.

**Comment by pat225** [comment]
In Menyoo I don´t see this option, only "Start Task Sequences Immediately" and in .xml in Spooner folder, I don´t see this option too. Where can I find this option? Thanks for help.

**Comment by OmegAlex10** [comment]
My bad, it's "start task sequences". It's how I call it since I mainly have animations as task sequences. Btw this is the case for reloaded files, if the file was already loaded and no animation played, then you have to add them again. (I just remembered that some animations ignore start task sequence immediately and must simply be set back on manually, I remember that happening to a paper clip animation)

**Comment by OmegAlex10** [comment]
I hope it helps

**Comment by pat225** [comment]
Yes, I want use clipboard animation and after load placement, animation is resetted. I must always set animations manually? It doesn´t work automatically?

**Comment by OmegAlex10** [comment]
Yep! It's incredibly annoying, but that's the only workaround I know

**Comment by OmegAlex10** [comment]
One more thing, maybe there is another clipboard animation that doesn't do that, you could experimemt with that and see which one isn't bugged

**Comment by pat225** [comment]
Can you help me, with other problem? I have for PED Task sequence and in task sequence I have "Play animation", but I have problem, I set animation durantion example, 20, it means, that animation have 20 seconds, but from some reason it is automatically setted on 0.0 second and animation doesn´t work. Why is in Task sequence animation time automatically resetted to 0.0 sec.?

**Comment by OmegAlex10** [comment]
Okkk.... Is the version that you currently have the latest release? (1.7 or something) Could be a bug at this point. Oh, and the scenario issues were a headache for a long time. People prefered soul stealing spawned peds and activating scenarios through simple trainer since menyoo's scenarios are annoying as hell to work with. (Disappearing, never appearing ever again even after a game restart, unless you make a new menyoo spooner save file). 
Last thing, I rarely use task sequence animations since they are tricky to time them (accurately by editng the save file's cfg or xml, I forgot.. I just browse through animations for each ped if their task is simple. I only use task sequences rarely for complex animation sequences when recording for like 5 or 10 seconds (preferably in one try)
Menyoo Spooner is sadly an incomplete mess..

**Comment by OmegAlex10** [comment]
To sum things up, scenarios are a broken mess on menyoo;(no matter the version), animations *mostly* work when chosen manually from ped settings on spooner mod, or it's just the latest version that's bugged

**Comment by pat225** [comment]
I have latest version, but I had this animation problem, that after Load placements, PEDs still without animation before too. It isn´t repaired. And, if I use simple trainer for scenarios, when I Load placements, scenarios are cancelled. I can´t problably combine this two mods. It is working only when I set it, but when I Load placements, they are cancelled.

**Comment by OmegAlex10** [comment]
Scenario's maybe require specific technicalities/a specific order like for example spawn with menyoo-spul steal-play animation through simple trainer-record-etc etc. Even if it doesn't load again, if it loads once (and can be repeated again) then you could definitely adapt to that workaround. I really don't know at this point, it's just a matter of workarounds.. That's all I know from personal experience and from tutorials

**Comment by pat225** [comment]
But I haven´t problem only with scenarios, animations doesn´t work too. I set some animation, save reference and than I Load placements and it doesn´t work too. It isn´t only scenarios, but Animations too.
